src/main/java/com/direwolf20/buildinggadgets/common/items/AbstractGadget.java

```diff
@@ -125,10 +125,10 @@ public boolean isDamageable() {
         return getMaxDamage() > 0;
     }
 
-    /*@Override
-    public boolean isRepairable() {
+    @Override
+    public boolean isRepairable(ItemStack stack) {
```

Comment:
You you look into the Block.Properties class and see if there is a property to set this in the constructor? This might be why I commented this out but might have used the wrong one. If you check Item class you should be able to see if this method uses one of those properties, if not then I'll accept :+1:

Reply:
I see what you mean, canRepair should do the trick.

Reply:
Yeah, maxDamage(1) on all the gadgets also needs to be fixed, this is likely why we have the durability issue as well as needing to set the canRepair prop.